packages/calcite-components/src/components/tab/tab.scss

```diff
@@ -3,7 +3,8 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-tab-content-block-padding: Specifies the block padding of the component's content in the `default` slot.
+ * @prop --calcite-tab-content-block-padding: [Deprecated] Use `--calcite-tab-content-space` instead. Specifies the block padding of the component's content in the `default` slot.
+ * @prop --calcite-tab-content-space: Specifies the space of the component's content in the `default` slot.
```

Comment:
Should this be “content block space”?

Reply:
Synced up w/ @alisonailea and landed on `--calcite-tab-content-space-y`. It's concise, readable and platform-agnostic.

Reply:
Do we need to update other space vars already merged?

Reply:
We should. I've got some block/block-section PRs to update. ✨💪✨